<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-SSDS6GEK" failures="2" tests="2" name="RetryMultiple" time="0.144" errors="0" timestamp="2023-08-03T22:55:16 EDT">
  <testcase classname="Retry.Practice" name="demo1" time="0.027">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Retry.Practice.demo1(Practice.java:9)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- demo1 -->
  <testcase classname="Retry.Practice" name="demo2" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Retry.Practice.demo2(Practice.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- demo2 -->
</testsuite> <!-- RetryMultiple -->
